plugins/generator-1.16.5/forge-1.16.5/procedures/_trim.java.ftl

```diff
@@ -0,0 +1 @@
+${opt.removeParentheses(value)}.trim()
```

Comment:
Surround the optimizer's method return value with brackets like it's done for MC 1.18. 2 and higher.